sql/lens/top10k/timeseries.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+  url,
+  _TABLE_SUFFIX AS _TABLE_SUFFIX
+FROM
+  `httparchive.summary_pages.*`
+WHERE
+  rank <= 10000 AND
```

Comment:
The only thing that changes per lens is this numeric value. Is there a way to reduce the duplication and dynamically change the rank value?

Reply:
Interesting… let me think on it.

Do agree there is duplication but on flip side it keeps complexity down in the main script itself and allows for different lenses in future (or keeping Wordpress/Drupal/Magento) without another if statement, so initial thoughts are to live with the duplication to be honest. But let me think it over…

Reply:
I'm going to leave this as is. Agree there is duplication, but quite a lot of of change required in the generate script to support this, and script is already getting a bit complex and difficult to maintain so don't think this is worth it.